examples/revpbe0-water-train-set/README.md

```diff
@@ -4,13 +4,13 @@ and solid water PNAS 2019
 
 #
 * step 1
-gen_soap_kmat.py -fxyz dataset_1593_eVAng.xyz --prefix H2O --rcut 4.0 --n 4 --l 6 --g 0.3 --periodic True --plot True
+python ../../scripts/gen_soap_kmat.py -fxyz dataset_1593_eVAng.xyz --prefix H2O --rcut 4.0 --n 4 --l 6 --g 0.3 --periodic True --plot True
```

Comment:
We decided to just use the name of the script, instead of the relative path in the examples,
because we expect people to install it